renaming_files.py

```diff
@@ -0,0 +1,50 @@
+'''Напишите функцию группового переименования файлов. Она должна:
+* принимать в качестве аргумента желаемое конечное имя файлов.
+* При переименовании в конце имени добавляется порядковый номер.
+* принимать в качестве аргумента расширение исходного файла.
+Переименование должно работать только для этих файлов внутри каталога.
+* принимать в качестве аргумента расширение конечного файла.
+Шаблон переименованного файла: <original_name>_<new_name>_<position>.<new_extention>'''
+import os
+
+
+def new_name_generator(name, count, new_name, new_extension):
+    temp_number = []
+    temp_count = list(str(count))
```

Comment:
Переводить строку в список не обязательно. Поведения кода от этого не поменяется.
```suggestion
    temp_count = str(count)
```